talking-about-their-pronouns.md

```diff
@@ -0,0 +1,38 @@
+# Asking someone's pronouns
+
+## Over Slack / in conversation
+
+What: Sorry, just to check so I can get them right, do you mind me asking what your pronouns are?
+
+# Correcting someone who misgenders someone else
+
+## During a meeting
+
+### Respond in the moment
+
+When: Once the person who's just got the pronouns incorrect has finished talking
+How: Verbally
+What: Just to point out, I'm pretty sure their pronouns are they/them
+
+### Respond async
+
+Where: Slack DM, email etc.
+What: FYI, I think you used "she" to refer to Sarah and I'm pretty sure their pronouns are they/them :)
+
+## Via Slack/email
+
+Where: In a direct message to that person (Slack DM, email, i.e. not in the initial thread itself to avoid derailing conversation if it happened in a group conversation)
+What: FYI, I'm pretty sure Sarah uses they/them pronouns :)
+
+# When you get pronouns wrong
+
+## If you realise during the interaction
+
+When: In speech, not as a big deal
```

Comment:
Maybe that's a good example to add in? People are likely to be better at that (/ I guess maybe find it less awkward?) so it might be a helpful mental model?

```
What: She, sorry, they

:bulb: In the same way that if you accidentally called someone the wrong name you'd quickly correct rather than making a big deal (e.g. `Nick, sorry, Rick`) you can correct yourself on pronouns too!